service.go

```diff
@@ -56,6 +56,37 @@ type IncidentUrgencyRule struct {
 	OutsideSupportHours *IncidentUrgencyType `json:"outside_support_hours,omitempty"`
 }
 
+// ListServiceRulesResponse represents a list of rules in a service
+type ListServiceRulesResponse struct {
+	Offset uint           `json:"offset,omitempty"`
+	Limit  uint           `json:"limit,omitempty"`
+	More   bool           `json:"more,omitempty"`
+	Total  uint           `json:"total,omitempty"`
+	Rules  []*ServiceRule `json:"rules,omitempty"`
```

Comment:
Yes, that sounds good. Just a quick question: I used a slice of pointers to keep it consistent with the `ListRulesetRulesResponse` struct, which is very similar to this one. It is fine to let them have different structures? 

The struct I am talking about: https://github.com/PagerDuty/go-pagerduty/blob/4d8ea2c7c6b4df8341901afb4b42b1460c75e2af/ruleset.go#L31-L38

Reply:
@mrzacarias That's a great question, and I'm sorry for not elaborating in my original comment. I want to change those too, but that would be a breaking change and so we can't do so in v1.

My goal is to do it in v2, and so since I know it'll be coming eventually I'm hoping to limit us adding more things we know we'll break in the future. Based on the changes you've made based on the feedback, I suspect your methods will make it into v2 with no changes required.

Thank you for that!